pages/common/nslookup.md

```diff
@@ -0,0 +1,27 @@
+# nslookup
+
+> Query name server(s) for various domain records.
+
+- Query your system default name server for an IP address (A record) of the domain:
+
+`nslookup  {{example.com}}`
+
+- Query non default name server (here Google DNS 8.8.8.8) for a NS record of the domain:
+
+`nslookup type=NS {{example.com}} 8.8.8.8`
+
+- Query default name server for an IP address (A record) of the domain using TCP protocol instead of the default UDP
+
+`nslookup -vc -type=A {{example.com}}`
+
+- Query default name server for a mail server (MX record) of the domain showing details of the transaction:
+
+`nslookup type=MX -debug {{example.com}}`
+
+- Query non default name server {{name_server}} on a non default port {{port_number}} for a TXT record of the domain:
+
+`nslookup  -port={{port_number}} -type=TXT {{example.com}} {{name_server}}`
+
+- Query default name server for a reverse lookup (PTR record) of an IP address:
+
+`nslookup -type=PTR 8.8.8.8`
```

Comment:
I suppose the 8.8.8.8 is the IP address here. Since it's used above as a name server, wouldn't it be less prone to confusion to use a different IP here?